tests/web3js/debug_traces_test.js

```diff
@@ -92,8 +93,8 @@ it('should retrieve transaction traces', async () => {
     // Assert proper response for `callTracer`
     txTrace = response.body.result
     assert.equal(txTrace.from, '0xfacf71692421039876a5bb4f10ef7a439d8ef61e')
-    assert.equal(txTrace.gas, '0x697f')
-    assert.equal(txTrace.gasUsed, '0x6827')
+    assert.equal(txTrace.gas, '0x6f9a')
+    assert.equal(txTrace.gasUsed, '0x6e3f')
```

Comment:
why is gasUsed changed?

Reply:
Because I also changed the contract call in the transaction, from `store` to `storeWithLog`, so that I can try out the `withLog` key in the tracer. Related change: https://github.com/onflow/flow-evm-gateway/pull/711/files#diff-92b8065b812d51562cf0af8e1389e757c9fd134b1c6d7f4c99afaee88e3f9366R74

Reply:
oh, ok, makes sense.